robot_description.py

```diff
@@ -438,5 +439,6 @@ def addJoint(self, jointType, linkFrom, linkTo, transform, name, jointLimits, zA
         # print('Joint from: '+linkFrom+' to: '+linkTo+', transform: '+str(transform))
     
     def finalize(self):
+        self.append(self.additionalXML)
```

Comment:
Note: the additional XML is IN the `<model></model>` for SDF, to mention in the README maybe